### ISSUE TITLE
Some questions about training

### ISSUE THREAD

**Issue by zhanjunlang** [gabrielStanovsky/supervised-oie#7]
Dear author, 

Firstly, is your training code in Allennlp open? In the <a href="https://allennlp.org/models#open-information-extraction"> link </a> you give, I can only find the prediction instruction and an online demo but without the training instruction.

Secondly, in your training data "train.oie.conll", some instances are exactly the same including the predicate except for their labels. Taking the example from your paper "Barack Obama, a former U.S president, was born in Hawaii.". It can generate 2 training instances with the same predicate "was born in" but with different A0 tags("Brarack Obama" and "a former U.S president"). According to your model, the inputs to the model of this 2 instances are exactly the same but you expect that it will output 2 different sets of label. Could you shed some light to this issue?

Many thanks!

**Comment by gabrielStanovsky** [comment]
Hi, 
Sorry for the late reply.

Yes, the training code is open at allennlp. You can see an example training configuration [here](https://github.com/allenai/allennlp/blob/master/allennlp/tests/fixtures/open_information_extraction/experiment.json)

Regarding different labels with the same predicate, can you please point me to an example? Thanks!
